gcp/appengine/frontend3/src/styles.scss

```diff
@@ -422,6 +422,53 @@ dl.vulnerability-details,
     grid-column: 2;
   }
 
+
+  dd.details {
+    overflow-wrap: break-word;
+    li {
```

Comment:
I wonder if the styles here and below we should be pulling out into a more top-level position.

Reply:
Hmm, this is to set specific styles for the user provided markdown, which probably don't make sense to be more top level? 

Do you have a suggestion for how to make these all cleaner?

Reply:
Good point - I felt like I saw duplication but when I took a closer look it's just this bit that I'm concerned with:

```css

    li {
      $active-spacing: 6px;
      margin: -$active-spacing 0;
      padding: $active-spacing 0;
    }
```

It's duplicated from `.top-bar` - this makes sense in a horizontal list of links context but doesn't seem to be styling that makes sense for typical paragraph content?

Reply:
removed as discussed!